PROTOCOL.md

```diff
@@ -965,6 +966,24 @@ class SetDirtyStateAction implements Action {
 ```
 </details>
 
+### 2.6.3. ExportSvgAction
+
+The client (or the server) sends a `ExportSvgAction` to indicate that diagram, which represents the current model state, should be exported in SVG format. The action provides the diagram svg as plain string.
+
+<details open><summary>Code</summary>
+
+```typescript
+class ExportSvgAction implements ResponseAction {
+    static KIND = 'exportSvg';
+    kind = ExportSvgAction.KIND;
+
+    constructor(public readonly svg: string,
```

Comment:
We usually do not provide the action constructor in the protocol but simply the commented (!), public fields which should be available. That you can use a constructor to automatically use public fields is a detail we omitted in the other actions.